egs/wsj/s5/steps/nnet3/lstm/decode.sh

```diff
@@ -30,6 +30,8 @@ minimize=false
 frames_per_chunk=10000
 extra_left_context=20   # it is recommended to use the same value as the chunk_left_context
                         # used during training
+extra_right_context=20  # it is recommended to use the same value as the chunk_right_context
```

Comment:
the default value has to be null string or 0, as the default recipe is lstm unless specified.